docs/connecting-to-a-lit-network/lit-blockchains/chronicle.md

```diff
@@ -18,11 +18,11 @@ Chronicle is Lit Protocol's custom EVM rollup testnet, which enhances the perfor
 
 Chronicle is a custom EVM rollup testnet based on the OP Stack, designed specifically for Lit Protocol. Anchored in the Polygon network, this rollup is the primary platform for coordination, minting PKPs (programmable key pairs), and managing PKP Permissions. Note that PKPs minted on Chronicle can still sign transactions on any chain supported by Lit, including EVM, Cosmos, and Bitcoin.
 
-## `testLPX` Test Token
+## `tstLPX` Test Token
 
-The `testLPX` test token serves as the gas for transactions on Chronicle. Please note that this is a test token with no real-world value. Its purpose is exclusively for testing and development on the Lit Protocol platform.
+The `tstLPX` test token serves as the gas for transactions on Chronicle. Please note that this is a test token with no real-world value. Its purpose is exclusively for testing and development on the Lit Protocol platform.
 
-To obtain the `testLPX` test token, please use the faucet at https://faucet.litprotocol.com/. The `testLPX` test token will be sent to your wallet address, allowing you to perform transactions on the rollup.
+To obtain the `tstLPX` test token, please use the faucet at https://faucet.litprotocol.com/. The `tstLPX` test token will be sent to your wallet address, allowing you to perform transactions on the rollup.
```

Comment:
faucet link should direct to Yellowstone

Reply:
Got all of the old faucet links